utils/stringUtils.js

```diff
@@ -46,7 +67,8 @@ const getTribunalPanel = ben => {
   return {
     PIP: 'judge, doctor and disability expert',
     DLA: 'judge and a doctor',
-    ESA: 'judge and a doctor'
+    ESA: 'judge and a doctor',
+    carersAllowance: 'judge and a doctor'
```

Comment:
I know not related to your PR, but what happens when benefit type is UC here?

Reply:
UPdated: I've added UC to this list and updated the template to use it accordingly so consistent with how the other benefits panel detaills are used here